packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -191,10 +191,7 @@ class GameWidgetState<T extends Game> extends State<GameWidget<T>> {
   Future<void> get loaderFuture => _loaderFuture ??= (() async {
         final game = currentGame;
         assert(game.hasLayout);
-        final onLoad = game.onLoadFuture;
```

Comment:
Is it safe to remove this call and rely only on super.load?

Sorry I am on mobile so would took me a while to check on the source of that method

Reply:
It's basically just a name change

Reply:
Hum, but is the onLoad method gone????

Reply:
Haha no, that would be a massive breaking change 😄
Only the internal `onLoadFeature` getter has been renamed to `load` and made into a method.

Reply:
Got ya! Phew, that massive breaking change was my concern haha